cli/utils/ts-linter.js

```diff
@@ -19,6 +19,10 @@ function lintSync() {
   logger.info('Starting TSLint...');
 
   const spawnResult = spawn.sync('./node_modules/.bin/tslint', flags);
+  if (spawnResult.error) {
```

Comment:
Thank you. I had to add this a few times (locally), but didn't get the opportunity to put it in a separate branch.